cfgen/cfgen

```diff
@@ -972,10 +972,11 @@ class ConfNode(ToDhall):
     _downlinks = {Downlink('processes', ObjT.process)}
 
     def __init__(self, name: str, nr_cpu: int, memsize_MB: int,
-                 processes: List[Oid]):
+                 processes: List[Oid], machine_id: str = None):
```

Comment:
If it is `str`, then `None` is an invalid value 😉 Can you double check why mypy didn't fail the build? Doesn't mypy check cfgen?

```suggestion
                 processes: List[Oid], machine_id: Optional[str] = None):
```

Reply:
> Can you double check why mypy didn't fail the build?

yes..Just checked.. mypy checks files with `.py` extensions.

Reply:
done